express-api/src/typeorm/Migrations/1717629146741-MoveMetadataToTables.ts

```diff
@@ -0,0 +1,165 @@
+import { In, MigrationInterface, QueryRunner } from 'typeorm';
+import { TimestampType } from '../Entities/TimestampType';
+import { ProjectStatus } from '../Entities/ProjectStatus';
+import { NoteType } from '../Entities/NoteType';
+import { MonetaryType } from '../Entities/MonetaryType';
+import { Task } from '../Entities/Task';
+import { User } from '../Entities/User';
+
+export class MoveMetadataToTables1717629146741 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const systemUser = await queryRunner.manager.findOne(User, { where: { Username: 'system' } });
+    const systemId = systemUser.Id;
+    const result = await queryRunner.manager.save(ProjectStatus, {
+      Id: 44, //For some reason there is no sequence on project_status
+      Name: 'Close Out',
+      Code: 'CLS-OUT',
+      Description: 'Status related to filling out the close out form.',
+      IsMilestone: true,
+      IsTerminal: true,
```

Comment:
Is it terminal? I'm not sure we're even using this column's values at the moment. If close out has to be done before the Disposal status, this could be an issue.

Reply:
I'll change this to be false for now, but yeah we aren't doing anything with this value on the new frontend at the moment.